src/games/index.js

```diff
@@ -0,0 +1,27 @@
+import { greet, checkAnswer } from '../cli.js';
+
+const engine = (game, description) => {
+  const username = greet();
+  let score = 0;
+  let defeat = false;
+  console.log(description);
+
+  while (score < 3 && !defeat) {
+    const { guess, correctAnswer } = game();
+    if (checkAnswer(guess, correctAnswer)) {
```

Comment:
I guess it better to rename `checkAnswer` to something like `isAnswerCorrect`, just because reads almost like a plain english: `if (isAnswerCorrect(quess, correctAnswer))`